nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java

```diff
@@ -221,6 +224,41 @@ public NullnessStore forceRunOnMethod(TreePath methodPath, Context context) {
     return dataFlow.finalResult(methodPath, context, nullnessPropagation);
   }
 
+  /**
+   * Nullness of the optional content of the expression is checked in the store.
+   *
+   * @param exprPath tree path of the expression
+   * @param context Javac context
+   * @return nullness info of optional content of the optional expression
+   */
+  public Nullness getNullnessOfExpressionOptionalContent(TreePath exprPath, Context context) {
+    NullnessStore store = dataFlow.resultBeforeExpr(exprPath, context, nullnessPropagation);
+
+    // We use the CFG to get the Node corresponding to the expression
+    Set<Node> exprNodes =
+        dataFlow
+            .getControlFlowGraph(exprPath, context, nullnessPropagation)
+            .getNodesCorrespondingToTree(exprPath.getLeaf());
+
+    if (exprNodes.size() != 1) {
+      // Since the expression must have a single corresponding node
```

Comment:
Am I missing any case in this? I assume that it will always have a single node in case of expression. If not, I'm returning the default value.

Reply:
I am not sure if this is true in all possible cases. So I took this safe approach. If this is true in all cases then we can change it. 
@msridhar  What do you think?

Reply:
I am not strongly against this one way or the other. On one hand, by crashing rather than potentially reporting a FP error, we signal to the user that it's NullAway itself that is missing support for this case. On the other, it's easier for users to suppress FPs and move on than work around a NullAway/compiler crash :)

Reply:
This looks fine to me